client/src/test/java/se/fortnox/reactivewizard/client/HttpClientTest.java

```diff
@@ -1140,6 +1142,26 @@ public void shouldHandleUnsafeSecureOnUntrustedHost() throws URISyntaxException
             .singleOrDefault(null);
     }
 
+    @Test
+    public void shouldLogRequestDetailsOnTimeout() {
+        HttpServer<ByteBuf, ByteBuf> server   = startServer(OK, Observable.never(), r->{});
+
+        try {
+            TestResource resource = getHttpProxy(server.getServerPort());
+            HttpClient.setTimeout(resource, 10, TimeUnit.MILLISECONDS);
+            resource.servertest("mode").toBlocking().single();
+            fail("expected timeout");
+        } catch (Exception e) {
+            OutputStream baos = new ByteArrayOutputStream();
+            PrintStream stream = new PrintStream(baos, true);
+            e.printStackTrace(stream);
```

Comment:
Forgotten debug? :)

Reply:
no, it writes to the stream that is used in assert below

Reply:
Ah, I was too quick :)